data/api.ts

```diff
@@ -12,6 +12,6 @@ export async function postEndpoint(url: string, data: any) {
   return fetchEndpoint(url, 'POST', data)
 }
 
-export async function putEndpoint(url: string, data: any) {
+export async function putEndpoint(url: string, data?: any) {
   return fetchEndpoint(url, 'PUT', data)
```

Comment:
After a recent change that has gone into `main` you should be able to omit the `data` param if it's not been provided. `fetchEndpoint` will then omit the `body` from the request if not provided.